client/src/App.svelte

```diff
@@ -148,9 +151,4 @@
   .chat--btn {
```

Comment:
Why not use `ml-5` instead ? If you start use tailwind and css both for same things (like spacing and sizing) it'll be harder to manage as styles get spread in multiple places